clients/src/main/java/org/apache/kafka/common/requests/FetchRequest.java

```diff
@@ -227,15 +234,13 @@ public static FetchRequest parse(ByteBuffer buffer, short version) {
     }
 
     @Override
-    protected Struct toStruct() {
-        short version = version();
-        Struct struct = new Struct(ApiKeys.FETCH.requestSchema(version));
+    public Struct toStruct() {
```

Comment:
I copy&pasted this from `ProduceRequest.java`. Let me change it to protected unless it doesn't work.